src/main/java/bio/terra/app/controller/DatasetsApiController.java

```diff
@@ -181,6 +182,18 @@ public ResponseEntity<DatasetModel> updateDatasetSnapshotBuilderSettings(
             id, userRequest, List.of(DatasetRequestAccessIncludeModel.SNAPSHOT_BUILDER_SETTINGS)));
   }
 
+  @Override
+  public ResponseEntity<SnapshotBuilderGetConceptsResponse> getConcepts(
+      @PathVariable("id") UUID id, @PathVariable("parentConcept") Integer parentConcept) {
```

Comment:
We're not consistent about this but in general it's not a good idea to copy down the parameter annotations from the openapi generated code. If they agree with the generated code, they have no effect, and if they disagree, it can lead to hard to find bugs.
```suggestion
      UUID id, Integer parentConcept) {
```